tests/unit/models/cart-model-test.js

```diff
@@ -312,8 +314,15 @@ test('it detects google analytics and appends the cross-domain linker param', fu
   assert.equal(model.checkoutUrl, `${baseUrl}/${variantId}:${quantity}?${query}`);
 
   const variant = singleProductFixture.product_listing.variants[1];
+  let globalScope;
```

Comment:
Don't duplicate this logic. Just include the global scope helper from `src/metal`